cqlsh_tests/test_cqlsh_copy.py

```diff
@@ -237,7 +237,10 @@ def all_datatypes_prepare(self):
                 u frozen<list<list<address_type>>>,
                 v frozen<map<map<int,int>,set<text>>>,
                 w frozen<set<set<inet>>>,
-                x map<text, frozen<list<text>>>
+                x map<text, frozen<list<text>>>,
+                y map<int, blob>,
+                z list<blob>,
+                aa set<blob>
```

Comment:
The columns need to be in alphabetical order, see 449e96b2a7aa8cc6dbbcf189e75b6f547e8e314f